param/serializer.py

```diff
@@ -25,37 +25,37 @@ class Serialization:
 
     @classmethod
     def schema(cls, pobj, subset=None):
-        raise NotImplementedError        # noqa: unimplemented method
+        raise NotImplementedError
```

Comment:
I'm unsure why this has changed; maybe it's because of  `--unsafe-fixes`. I don't think they should be added back

Reply:
Ruff complained that specific rules should be used with noqa. So I removed them. Forgot to explain that.